db/migrate/20221017162852_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Please edit this reference because it's supposed to be `author_id` not `user_id` as a foreign key.
[Hint] You can do it like that:
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```